pymmcore_plus/core/_property.py

```diff
@@ -0,0 +1,145 @@
+from typing import Any, Optional, Tuple
+
+from pymmcore import g_Keyword_Label, g_Keyword_State
+
+from ._constants import DeviceType, PropertyType
+from ._mmcore_plus import CMMCorePlus
+
+
+class MMProperty:
+    """Convenience "View" onto a device property.
+
+    Parameters
+    ----------
+    device_label : str
+        Device this property belongs to
+    property_name : str
+        Name of this property
+    mmcore : Optional[CMMCorePlus]
+        CMMCore instance, by default global singleton.
+
+    Examples
+    --------
+
+    >>> prop = MMProperty('Objective', 'Label')
+    >>> prop.dict()
+    >>> prop.core.loadSystemConfiguration()
```

Comment:
I'm not sure i love this access pattern. I can see having it be allowed on this object but maybe we should try to always use `instance()` in examples?

Reply:
can remove.  Ultimately, this class is just a combination of a core instance, and two strings.  I think it makes sense to be able to access and  set the core instance, but this particular example "sneakily" access the global singleton.  Is it just that?  or is it the fact that you can get `prop.core` at all?

Reply:
It's just the sneaky access that was worrying me. The functionality of accessing it makes sense

Reply:
Then I think we should make core a mandatory parameter, and remove `self._mmc = mmcore or CMMCorePlus.instance()` from line 46